dotnet/Dockerfile.tpl

```diff
@@ -0,0 +1,27 @@
+# AUTOGENERATED FILE
+FROM #{FROM}
+
+RUN [ "cross-build-start" ]
+
+RUN apt-get update && \
+    apt-get install -y curl libunwind8 gettext icu-devtools
+
+RUN rm -rf /tmp/* /var/tmp/* && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin
+
+WORKDIR /tmp
+
+RUN curl -sSL -o dotnet.tar.gz #{DOTNET_BUNDLE}
```

Comment:
Can you add the shasum check to verify the file integrity here, like this https://github.com/resin-io-library/base-images/blob/master/node/raspberrypi3/debian/9.11/Dockerfile#L14-L17?